scootapi/scoot.thrift

```diff
@@ -55,7 +55,7 @@ struct TaskDefinition {
 
 struct JobDefinition {
   1: required list<TaskDefinition> tasks,
-  2: optional JobType jobType,
+  2: optional JobType DEPRECATED_jobType,
```

Comment:
We can remove this after everything has been redeployed (but leave something to indicate that field '2:' is no longer available.)